Ground-Items.md

```diff
@@ -118,20 +121,42 @@ Configures the color for insane value items.
 
 Configures the start price for insane value items.
 
-### Only show loot
+### Only show own items
 
-Only shows drops from NPCs and players.
+Only shows items that are yours or you can pick up.
```

Comment:
Not 100% true rn since it was nerfed again because of people complaining. World spawns and maybe your own arrows will not be visible, but irons can pick them up.